CFC_WebApp/main/get_database.py

```diff
@@ -37,10 +39,37 @@ def get_profile_db():
     Profiles=current_db.Stage_Profiles
     return Profiles
 
+"""
 def get_routeDistanceMatrix_db():
     current_db=MongoClient().Stage_database
     routeDistanceMatrix=current_db.Stage_routeDistanceMatrix
     return routeDistanceMatrix
+"""
+
+def get_routeDistanceMatrix_db(user_id, method):
+    if not os.path.exists('routeDistanceMatrices'):
+        os.makedirs('routeDistanceMatrices')
+    
+    routeDistanceMatrix = {}
+
+    user_id = str(user_id)
+    if not os.path.exists('routeDistanceMatrices/' + user_id + '_' + method + '_routeDistanceMatrix.json'):
+        data = {}
+        f = open('routeDistanceMatrices/' + user_id + '_' + method + '_routeDistanceMatrix.json', 'w+')
+        f.write(json.dumps({}))
+        f.close()
+    else:
+        f = open('routeDistanceMatrices/' + user_id + '_' + method + '_routeDistanceMatrix.json', 'r')
+        routeDistanceMatrix = json.loads(f.read())
+    return routeDistanceMatrix
+
+def update_routeDistanceMatrix_db(user_id, method, updatedMatrix):
```

Comment:
Its meant to serve the functionality of updating. It overwrites as of now but should not be renamed as such, because an ideal implementation would append rows and columns to the end.

Sent from my iPhone

> On May 6, 2015, at 1:02 PM, shankari <notifications@github.com> wrote:
> 
> In CFC_WebApp/main/get_database.py:
> 
> > +        os.makedirs('routeDistanceMatrices')
> > +    
> > +    routeDistanceMatrix = {}
> > +
> > +    user_id = str(user_id)
> > +    if not os.path.exists('routeDistanceMatrices/' + user_id + '_' + method + '_routeDistanceMatrix.json'):
> > +        data = {}
> > +        f = open('routeDistanceMatrices/' + user_id + '_' + method + '_routeDistanceMatrix.json', 'w+')
> > +        f.write(json.dumps({}))
> > +        f.close()
> > +    else:
> > +        f = open('routeDistanceMatrices/' + user_id + '_' + method + '_routeDistanceMatrix.json', 'r')
> > +        routeDistanceMatrix = json.loads(f.read())
> > +    return routeDistanceMatrix
> > +
> > +def update_routeDistanceMatrix_db(user_id, method, updatedMatrix):
> @jeffdh5, is this update or overwrite?
> 
> —
> Reply to this email directly or view it on GitHub.
>